plugins/GSdx/Window/GSSetting.cpp

```diff
@@ -92,9 +92,9 @@ const char* dialog_message(int ID, bool* updateText) {
 		case IDC_OSD_MONITOR:
 			return "Continuously prints the FPS and the EE, GS and VU(if the MTVU speedhack is enabled) percentages onscreen.";
 		case IDC_PALTEX:
-			return "When checked 4/8 bits texture will be send to the GPU with a palette. GPU will be in charge of the conversion.\n\n"
-				"When unchecked the CPU will convert directly the texture to 32 bits.\n\n"
-				"It is basically a trade-off between GPU/CPU.";
+			return "Enabled: GPU will handle 4/8 bits textures while CPU does the rest.\n"
```

Comment:
I would want a GS expert to proof this one but to me this doesn't sound right. To me, what I think it should be:

Enabled: GPU will handle conversion of 4 and 8 bit textures to 32 bit.
Disabled: CPU will handle conversion of 4 and 8 bit textures to 32 bit.

Reply:
Also the reason i want to change the tooltips because they even confuse me.